modules/UpgradeWizard/silentUpgrade_step2.php

```diff
@@ -308,17 +308,7 @@ function rebuildRelations($pre_path = '')
     exit(1);
 }
 
-$unzip_dir = sugar_cached("upgrades/temp");
-
-include "$unzip_dir/manifest.php";
-if (is_file("$unzip_dir/manifest.php")) {
```

Comment:
perhaps this was meant to be !is_file ??

anyway is it the same issue in modules/UpgradeWizard/silentUpgrade_step1.php:545